iset.mm

```diff
@@ -63996,31 +64051,29 @@ all reals (greatest real number) for which all positive reals are greater.
   ${
     $d X x y $.  $d A x y $.
     djuf1olem.1 $e |- X e. _V $.
-    djuf1olem.2 $e |- F = ( x e. _V |-> <. X , x >. ) $.
-    $( Lemma for ~ djulf1oN and ~ djurf1oN .  Remark: maybe a version of this
-       lemma with ` F ` defined on ` A ` and no restriction in the conclusion
-       would be more usable.  TODO: add a version of ~ elimag using function
-       notation ` ( F `` A ) ` .  (Contributed by BJ and Jim Kingdon,
+    djuf1olem.2 $e |- F = ( x e. A |-> <. X , x >. ) $.
+    $( Lemma for ~ djulf1o and ~ djurf1o .  (Contributed by BJ and Jim Kingdon,
        4-Jul-2022.) $)
-    djuf1olemN $p |- ( F |` A ) : A -1-1-onto-> ( { X } X. A ) $=
-      ( vy csn cxp cres wf1o wtru cv cop cfv cvv cmpt wceq wcel adantl c2nd wss
-      reseq1i ssv resmpt ax-mp eqtri snid opelxpi mpan xp2nd c1st 1st2nd2 xp1st
-      wb elsni syl opeq1d eqtrd eqeq2d eqcom eqid opth mpbiran 3bitr3g ad2antll
-      vex bicomd f1o2d trud ) BDHZBIZCBJZKLAGBVLDAMZNZGMZUAOZVMVMAPVOQZBJZABVOQ
-      ZCVRBFUCBPUBVSVTRBUDAPBVOUEUFUGVNBSZVOVLSZLDVKSWAWBDEUHDVNVKBUIUJTVPVLSZV
-      QBSLVPVKBUKTWCVNVQRZVPVORZUOLWAWCWEWDWCVOVPRVODVQNZRZWEWDWCVPWFVOWCVPVPUL
-      OZVQNWFVPVKBUMWCWHDVQWCWHVKSWHDRVPVKBUNWHDUPUQURUSUTVOVPVAWGDDRWDDVBDVNDV
-      QEAVGVCVDVEVHVFVIVJ $.
+    djuf1olem $p |- F : A -1-1-onto-> ( { X } X. A ) $=
+      ( vy csn cxp wf1o wtru cv cop c2nd cfv wcel snid opelxpi adantl wceq mpan
+      xp2nd wb c1st 1st2nd2 xp1st elsni syl opeq1d eqtrd eqeq2d eqcom eqid opth
+      vex mpbiran 3bitr3g bicomd ad2antll f1o2d trud ) BDHZBIZCJKAGBVCDALZMZGLZ
+      NOZCFVDBPZVEVCPZKDVBPVHVIDEQDVDVBBRUASVFVCPZVGBPKVFVBBUBSVJVDVGTZVFVETZUC
+      KVHVJVLVKVJVEVFTVEDVGMZTZVLVKVJVFVMVEVJVFVFUDOZVGMVMVFVBBUEVJVODVGVJVOVBP
+      VODTVFVBBUFVODUGUHUIUJUKVEVFULVNDDTVKDUMDVDDVGEAUOUNUPUQURUSUTVA $.
+  $}
 
-    $( Lemma for ~ djulf1o and ~ djurf1o .  (Contributed by BJ and Jim Kingdon,
+  ${
+    $d X x y $.  $d A x y $.
+    djuf1olemr.1 $e |- X e. _V $.
+    djuf1olemr.2 $e |- F = ( x e. _V |-> <. X , x >. ) $.
+    $( Lemma for ~ djulf1oN and ~ djurf1oN .  Remark: maybe a version of this
+       lemma with ` F ` defined on ` A ` and no restriction in the conclusion
+       would be more usable.  (Contributed by BJ and Jim Kingdon,
        4-Jul-2022.) $)
-    djuf1olem $p |- F : _V -1-1-onto-> ( { X } X. _V ) $=
-      ( vy cvv csn cxp wf1o wtru cv cop c2nd cfv wcel snid opelxpi adantl wceq
-      mpan xp2nd wb c1st 1st2nd2 xp1st elsni syl opeq1d eqtrd eqeq2d eqcom eqid
-      vex opth mpbiran 3bitr3g bicomd ad2antll f1o2d trud ) GCHZGIZBJKAFGVCCALZ
-      MZFLZNOZBEVDGPZVEVCPZKCVBPVHVICDQCVDVBGRUASVFVCPZVGGPKVFVBGUBSVJVDVGTZVFV
-      ETZUCKVHVJVLVKVJVEVFTVECVGMZTZVLVKVJVFVMVEVJVFVFUDOZVGMVMVFVBGUEVJVOCVGVJ
-      VOVBPVOCTVFVBGUFVOCUGUHUIUJUKVEVFULVNCCTVKCUMCVDCVGDAUNUOUPUQURUSUTVA $.
+    djuf1olemr $p |- ( F |` A ) : A -1-1-onto-> ( { X } X. A ) $=
```

Comment:
What does the suffix "r" mean?

Reply:
"restriction". I'm open to suggestions.

Reply:
Since this is just the name of a lemma, I'm fine with anything. I mean, if you want it to be descriptive, you'd say `res` but I don't really expect that of lemmas (in fact the number one reason I usually name them instead of numbering them is that I often add/remove them while writing a proof and don't feel like renumbering each time).